### PR TITLE
197 refactoring

### PR DESCRIPTION
**Pull request by CaluvH**
Closing #197

### CODE REVIEW

File: StayAtHome-Client/Assets/Scripts/DataContainer/PlantData.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace DataContainer
+{
+    /// <summary>
+    /// Struct to store all plant related data.
+    /// It acts as a container for save and load operations w/o the need to instantiate the whole derived APlant-class
+    /// </summary>
+    public class PlantData
```
Comment (prooxeydev) [inline StayAtHome-Client/Assets/Scripts/DataContainer/PlantData.cs]: Why is this a class?

Comment (CaluvH) [reply StayAtHome-Client/Assets/Scripts/DataContainer/PlantData.cs]: This class is used to capsule the data of a plant. When the data is capsulated we can easily used it during saving and loading the data via the SavegameManager without initializing the final Plant-Object.

Comment (prooxeydev) [reply StayAtHome-Client/Assets/Scripts/DataContainer/PlantData.cs]: I just meant why is this a class, not a struct?

Comment (CaluvH) [reply StayAtHome-Client/Assets/Scripts/DataContainer/PlantData.cs]: Good question. I will change it to a strict ;)

**Pull request merged**